### PR TITLE
unquote paths on Windows

### PR DESCRIPTION
**Pull request by elharo**
@lesv @chanseokoh @loosebazooka

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java [hunk 1]
```diff
@@ -111,6 +117,17 @@ private static String getLocalAppDataLocation() {
     return possiblePaths;
   }
 
+  @VisibleForTesting
+  static String unquote(String path) {
+    if (path.startsWith("\"")) {
```
Comment (chanseokoh) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java]: Isn't it safer to combine the two conditions?

`if (path.startsWith("\"") && path.endsWith("\""))`

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java [hunk 2]
```diff
@@ -84,12 +85,17 @@ private static String getLocalAppDataLocation() {
...
-  private static List<String> getLocationsFromPath() {
+  @VisibleForTesting
+  static List<String> getLocationsFromPath(String pathEnv) {
     List<String> possiblePaths = new ArrayList<>();
-    String pathEnv = System.getenv("PATH");
 
     if (pathEnv != null) {
       for (String path : pathEnv.split(File.pathSeparator)) {
+        // Windows sometimes quotes paths so we need to strip these. 
+        // However quotes are legal in Unix paths.
```
Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java]: Not sure if you mean *illegal* in Unix paths

Comment (elharo) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java]: Nope, legal is correct. " is an acceptable character in Unix file names.

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java (hunk 1 above)
Comment (elharo) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java]: Absent normative documentation, I don't know that it is. The " is always illegal in paths on Windows, even if there's only one.

**Pull request merged**